common/src/crt0-vexriscv.S

```diff
@@ -34,6 +37,78 @@ _start:
 .section .text
 .global  trap_entry
 trap_entry:
+#ifdef CONFIG_CPU_VARIANT_FOMU
```

Comment:
Perhaps define a different var that is more descriptive of why we need to use `instruction_handler`, e.g. `CONFIG_DIV_UNIMPLEMENTED`?    We could set it in litex/soc/cores/cpu/vexriscv/core.py when the Fomu variant is chosen, using `soc.add_config("DIV_UNIMPLEMENTED")`.

Reply:
Yeah that seems like a better approach. I've created a [PR in your fork of LiteX](https://github.com/tcal-x/litex/pull/2), once that's merged I'll update the crt0 in this PR to use `CONFIG_CPU_DIV_UNIMPLEMENTED`.

Reply:
The `#ifdef` has been updated to be conditional on `CONFIG_CPU_DIV_UNIMPLEMENTED`, thanks again for the great suggestion!